usr/local/share/bastille/version.sh

```diff
@@ -0,0 +1 @@
+BASTILLE_VERSION="0.11.20241023"
```

Comment:
Could also add a comment there to explain that changes trigger a CI release flow.